sonar-db/src/main/resources/org/sonar/db/ce/CeActivityMapper.xml

```diff
@@ -18,12 +18,26 @@
     ca.updated_at as updatedAt,
     ca.is_last as isLast,
     ca.is_last_key as isLastKey,
-    ca.execution_time_ms as executionTimeMs
+    ca.execution_time_ms as executionTimeMs,
+    ca.error_message as errorMessage,
+    <choose>
```

Comment:
Ohoh, now I understand !
You could have added a new column has_error_stack_trace but I imagine you find it overkill...